### ISSUE TITLE
Edit button for translation files

### ISSUE THREAD

**Issue by matchachoco95** [tls-kr/tls-app#140]
Hello! 

As already mentioned in my last e-mail to @cwittern, a somewhat minor feature I would like to see implemented is an edit button for translation files. At the moment, it is possible to create new files and edit the lines of translation in them but it is still not possible to rename, add to, or otherwise edit the already existent file itself. Now that an option like this was added for the bibliographic references (thanks again!), it would be nice to have it for the translation files as well, since, if you've made a mistake in the creation, there is currently no other option than to create another file with the same or a very similar name, causing confusion and unnecessarily cluttering the system. 

As for where that edit button could go, I was thinking that maybe that could be handled by, for example, adding a small pen symbol next to the individual translation files of a text - or perhaps one could have all the files one created listed in one's account information somewhere and have the option to change them there so no one messes with another's file.

Thanks in advance!

**Comment by cwittern** [comment]
Thanks for this suggestion.  I am looking into it.  In keeping with the general policy of the TLS, I think members of the `tls-editor` group and the original creator should be allowed to change the metadata for a translation file.  Should there  be a way to delete the file?  Of course only if it has no translation lines yet.

**Comment by matchachoco95** [comment]
Yes, agreed - it would make sense if people from the group of editors also had access, for example in cases where the original creator (if they're not from that group) is having trouble working things out; they could then just reach out to someone with a more "powerful" account and have them help. 

Deleting would be a nice option too, yes, and I believe it's a good idea to limit that to "empty" translation files so nothing gets lost by accident. Thank you!

**Comment by cwittern** [comment]
This is live now on the system, except for the deleting, which will come later.

**Comment by matchachoco95** [comment]
Thank you so much! I just checked and unfortunately, it does not seem to be working for me as of now; when I click on the button for "more information", nothing happens. Also - and this is purely for aesthetic reasons - I think it would be nice if the dark grey box in the background looked unified, without those edges in it (see screenshot). 

![01 06 23](https://github.com/tls-kr/tls-app/assets/116962137/744df86a-bd76-4ac0-9dc2-bdb61f148656)

**Comment by cwittern** [comment]
Well, the rounded corners are by design, we not going to change that.  As to the button not working, did you try to force reload?  Hold the shift key and press the reload button on your browser.   (This is necessary, because some of the files the browser usually caches and does not request from the server have been changed)
